modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/ElementSetList.es.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {EmptyState, FieldType, SearchUtils} from 'data-engine-taglib';
+import {
+	elementSetAdded,
+	useForm,
+	useFormState,
+} from 'dynamic-data-mapping-form-renderer';
+import React from 'react';
+
+const EmptyPanel = ({searchTerm}) => (
+	<div className="mt-2">
+		<EmptyState
+			emptyState={{
+				description: Liferay.Language.get(
+					'there-are-no-element-sets-yet'
+				),
+				title: Liferay.Language.get('there-are-no-element-sets'),
+			}}
+			keywords={searchTerm}
+			small
+		/>
+	</div>
+);
+
+const ElementSetList = ({elementSets, searchTerm = '', ...context}) => {
+	const {activePage, pages} = useFormState();
+	const regex = SearchUtils.getSearchRegex(searchTerm);
+	const elementSetList = elementSets.filter(({name}) => regex.test(name));
+	const dispatch = useForm();
```

Comment:
hey @alinedoleron and @miltonmc, Normally we have a rule that if all the constants are declared together they must follow the alphabetical order if a constant depends on each other, a line break must be added. In my point of view, this is not very critical here or prevents sending this PR forward.